cmd/util/config.go

```diff
@@ -0,0 +1,71 @@
+// This package is for command line specific utilities.
+package util
```

Comment:
I feel this configuration object will be needed in other modules, so I believe it should be in a generic package, easily accessible to other modules.